front/components/agent_builder/instructions/AgentBuilderInstructionsEditor.tsx

```diff
@@ -135,8 +135,21 @@ export function AgentBuilderInstructionsEditor({
         suggestion: suggestionHandler,
       }),
       Placeholder.configure({
-        placeholder:
-          "What does this agent do? How should it behave? What should it avoid doing?",
+        placeholder: ({ editor }) => {
+          // Don't show placeholder inside instruction blocks
+          const { state } = editor;
+          const { selection } = state;
+          const $pos = selection.$anchor;
+
+          // Check if we're inside an instruction block
+          for (let depth = $pos.depth; depth > 0; depth--) {
```

Comment:
I don't really like this and I think the correct way is only show it for the first paragraph but I cannot make it work, I will see if I can fix it tonight  https://tiptap.dev/docs/editor/extensions/functionality/placeholder#additional-setup